qa/rpc-tests/test_framework/util.py

```diff
@@ -129,17 +129,19 @@ def sync_blocks(rpc_connections, *, wait=1, timeout=60):
     one node already synced to the latest, stable tip, otherwise there's a
     chance it might return before all nodes are stably synced.
```

Comment:
Does this comment still apply, considering that you switched from `0` to `getblockcount`?